shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/util/TableExtractUtils.java

```diff
@@ -198,7 +198,12 @@ public void extractTablesFromUpdate(final UpdateStatement updateStatement) {
         }
     }
     
-    private boolean needRewrite(final OwnerSegment owner) {
+    /**
```

Comment:
Util class should be final and static

Reply:
This util have state,can not call it directly by classname.

Reply:
Ok,I'll change the class name.